lib/correios_sigep/configuration.rb

```diff
@@ -1,12 +1,12 @@
 module CorreiosSigep
   class Configuration
-    attr_accessor :administrative_code, :card, :contract, :password,
-                  :service_code, :user, :wsdl_base_url, :proxy, :timeout
+    attr_accessor :administrative_code, :card, :service_code, :wsdl_base_url, :proxy, :timeout,
+      :user, :password
```

Comment:
Can u check how the styleguide handles this?
I think it asks to ident the attributes with the first parameter itself.